Library/Homebrew/formula.rb

```diff
@@ -922,6 +897,33 @@ def plist_startup
     self.class.plist_startup
   end
 
+  # This method can be used to provide a plist.
```

Comment:
Can you move this method to the same location as the old one? It makes seeing the diff easier.